src/components/daySelector/_tests_/daySelector.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import DaySelector from '../DaySelector';
+import { Day } from '../../../models/app.model';
+
+describe('DaySelector', () => {
+  const daysOfWeek: Day[] = [Day.Sun, Day.Mon, Day.Tue, Day.Wed, Day.Thu, Day.Fri];
+
+  test('renders days correctly', () => {
+    const { getByText } = render(
+      <DaySelector daysOfWeek={daysOfWeek} selectedDay={null} onSelectDay={() => {}} />
+    );
+
+    daysOfWeek.forEach((day) => {
+      const abbreviatedDay = day.substring(0, 3);
+      expect(getByText(abbreviatedDay)).toBeDefined();
```

Comment:
NGL, I don't understand what this tests

Reply:
on the day-menu the days are displayed "sun, mon" etc. 
this test ensures that each day of the week (abbreviated) is present in the rendered component as sunday, monday etc. If any day is missing or misspelled, the test will fail.